extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/interceptor/TransactionalInterceptorBase.java

```diff
@@ -227,10 +252,18 @@ protected Object invokeInCallerTx(InvocationContext ic, Transaction tx) throws E
     }
 
     protected Object invokeInNoTx(InvocationContext ic) throws Exception {
-
+        checkConfiguration(ic);
```

Comment:
Is it me or does this emit the RTE when there is noTX annotation and a timeout is set, which might be valid to do but doesn't match the RTE exception message

Reply:
`invokeInNoTx` is invoked when there is a `@Transactional` annotation with a `TxType` of `NOT_SUPPORTED` or `NEVER` or with `SUPPORT` for new transactions.

When there is no transaction annotation, the interceptor will not be called at all.

The RTE should occurs at development time to give the developer a clue that he misuse it so it should remove the annotation or move it to the right place.

Reply:
I see - thanks for confirming and seems OK to me. But the message you would receive in that circumstance is not broad enough as I think that "Cannot change timeout via @TransactionConfiguration in an already started transaction" is not an accurate summary of the state of the application for the case of NEVER? - WDYT?

Reply:
What about
> Cannot change timeout via @TransactionConfiguration in an already started transaction or when no transaction is supported
Or more simple
> Changing timeout via @TransactionConfiguration can only be done at the entry level of a transaction

Maybe you have a better wordings to propose :)

Reply:
I think the second one covers both and is more succinct - thanks!

Reply:
done

Reply:
LGTM - thanks! resolved from my POV